apps/wallet/src/ui/app/components/menu/content/MenuListItem.tsx

```diff
@@ -21,15 +22,13 @@ function MenuListItem({
     subtitle,
     iconAfter,
     to = '',
+    href = '',
     onClick,
 }: ItemProps) {
-    const Component = to ? Link : 'div';
-    return (
-        <Component
-            className="flex flex-nowrap items-center px-1 py-4.5 first:pt-3 last:pb-3 gap-5 no-underline overflow-hidden group cursor-pointer"
-            to={to}
-            onClick={onClick}
-        >
+    const Component = to || href ? Link : 'div';
```

Comment:
it looks like we're not using `Component` if we have `href`? was that intentional?

Reply:
Let me remove `href`